src/Core/Webservice/ServerRequirementsChecker.php

```diff
@@ -149,7 +149,7 @@ private function getWarningMessages()
     {
         return [
             self::ISSUE_APACHE_MOD_AUTH_BASIC_NOT_AVAILABLE => $this->translator->trans(
-                'Please activate the \'mod_rewrite\' Apache module to allow the PrestaShop webservice.',
+                'Please activate the \'mod_auth_basic\' Apache module to allow the use of the PrestaShop webservice.',
```

Comment:
Hi @kpodemski 

If this wording appears in the installer, then the translation domain should be "Install". :-)

Reply:
Hello @Julievrz 

Could you tell me the entire domain that you would suggest? Right now, all wording in the installer is within the same domain.

https://github.com/PrestaShop/PrestaShop/blob/451b19538e51f434f4d5bcd266f122f64fbe723f/src/Core/Webservice/ServerRequirementsChecker.php#L150

Reply:
Actually, the full domain is "Install". 😊 You can have a look here: https://zeroheight.com/80a6c5a61/v/latest/p/738e72-manage-translation-domains

Reply:
I've just changed the domain to `Install`, thanks 👍